### PR TITLE
STAND-103:Update CIEL version in the standalone pom

### PR DESCRIPTION
**Pull request by gitcliff**
LINK:https://issues.openmrs.org/browse/STAND-103

### CODE REVIEW

**Comment by gitcliff** [comment]
cc @dkayiwa

File: pom.xml [hunk 1]
```diff
@@ -12,9 +12,10 @@
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<openmrs.version>2.3.0</openmrs.version>
```
Comment (dkayiwa) [inline pom.xml:15]: Why the above version?

Comment (gitcliff) [reply pom.xml:15]: @dkayiwa  i wanted to depend on a recent stable version of openmrs which is already released and in production

Comment (dkayiwa) [reply pom.xml:15]: Is that the most recent release?

**Comment by gitcliff** [comment]
@dkayiwa i have adjusted it to `2.3.1`

**Pull request merged**